waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -335,7 +359,9 @@ proc connectToNodes*(pm: PeerManager,
 
 # Ensures a healthy amount of connected relay peers
 proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
-  while true:
+  pm.relayLoopUp = true
+  info "Starting relay connectivity loop"
```

Comment:
dont hold a strong opinion so can move to debug. but my rationale behind this:

both `Starting relay connectivity loop` and `Starting service connectivity loop` are a one time thing, and mimic "mounting relay protocol", "mounting libp2p ping protocol", "starting relay protocol", etc, which i think are quite relevant.

I also think `Adding peer to service slots` is quite relevant since it shows the service peers we are using, and the impact of this i quite high (eg in store). As it is is a one time thing (taken from cli flags).

Regarding logging connected peers every few seconds, sure I can use this metrics dump every minute for that. iirc the connected peers metrics is not there (just the delta aka new connection since last cycle)?

Reply:
done! using debug now